servalcat/spa/fofc.py

```diff
@@ -377,6 +377,17 @@ def write_coot_script(py_out, model_file, mtz_file, contour_fo=1.2, contour_fofc
                 ofs.write("add_molecular_symmetry(imol, {})\n".format(",".join(str(x) for x in v)))
 # write_coot_script()
 
+def write_chimerax_script(cxc_out, model_file, fo_mrc_file, fofc_mrc_file):
+    with open(cxc_out, "w") as ofs:
+        ofs.write('open {}\n'.format(model_file))
+        ofs.write('open {}\n'.format(fo_mrc_file))
+        ofs.write('open {}\n'.format(fofc_mrc_file))
+        ofs.write('volume #3 level 4 level -4 color #00FF00 color #FF0000 squaremesh false cap false style mesh meshlighting false\n')
```

Comment:
I think this line should be moved to the end, after both `clipper associate` commands. Because `clipper associate` will change the display style, so these customization will be lost if they run before `clipper associate`.